stockflux-watchlist/src/App.js

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
-import Watchlist from './components/Watchlist';
-import Components from 'stockflux-components';
+import React from "react";
+import { Watchlist } from "./components/Watchlist/Watchlist";
```

Comment:
It seems that `WatchList` is the default export, so you could remove the `{` and `}` here

Reply:
Indeed, thanks for spotting! Fixed -> 13d11dfc1caba9c5e79b0182a6313153503d112e